### ISSUE TITLE
Proxmox v0.0.4 _DoCommand, No entities or services

### ISSUE THREAD

**Issue by jamman9000** [kevinvincent/ha-wyzesense#19]
Hello, I've been playing with this for a few days, asking around and trying different things. I successfully passed the USB into my hassio VM (install from the one-click proxmox script), and when I run ```dmesg | grep hidraw``` I get:

```core-ssh:/config# dmesg | grep hidraw
[    1.130239] hidraw: raw HID events driver (C) Jiri Kosina
[    2.056073] hid-generic 0003:0627:0001.0001: input,hidraw0: USB HID v0.01 Mouse [QEMU QEMU USB Tablet] on usb-0000:00:01.2-1/input0
[    2.529353] hid-generic 0003:1A86:E024.0002: hiddev96,hidraw1: USB HID v1.00 Device [HID 1a86:e024] on usb-0000:00:1d.0-2/input0
```

Configuration.yaml is:
```binary_sensor:
  - platform: wyzesense
    device: "/dev/hidraw1"
```
I first tried with ```hidraw0```, then with ```hidraw1```. I have a blue light on my Wyze Sense hub and logs seem to show activity but then fail. Please see this pastebin, note the comments about starting with v0.0.4, downgrading to v0.0.3 to test, and then back up to v0.0.4:
[Pastebin Link](https://pastebin.com/rPZ2szJ2)

**Comment by kevinvincent** [comment]
@jamman9000 Looks like 3 or 4 reports came in of that same issue after .4. I will investigate tonight and hopefully have a fix.

**Comment by jamman9000** [comment]
Any luck with this?

**Comment by kevinvincent** [comment]
Can you try upgrading to 0.0.5 and see if the issue still persists? I'm not completely sure what this could be after digging into it. However, I have an overall larger change im working on that should also help with this and similar issues. It's more involved though so it is taking a while.

**Comment by jamman9000** [comment]
Ok, looking forward to it. I upgraded and get a new error now:

Fri Aug 30 2019 09:04:13 GMT-0400 (Eastern Daylight Time)
Error doing job: Future exception was never retrieved
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/concurrent/futures/thread.py", line 57, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/config/custom_components/wyzesense/binary_sensor.py", line 85, in setup_platform
    result = ws.List()
  File "/config/custom_components/wyzesense/wyzesense_custom.py", line 507, in List
    sensors = self._GetSensors()
  File "/config/custom_components/wyzesense/wyzesense_custom.py", line 479, in _GetSensors
    self._DoCommand(Packet.GetSensorList(count), cmd_handler, timeout=self._CMD_TIMEOUT * count)
  File "/config/custom_components/wyzesense/wyzesense_custom.py", line 384, in _DoCommand
    raise TimeoutError("_DoCommand")
TimeoutError: _DoCommand

**Comment by jamman9000** [comment]
Update: While I am getting that error, and the wyzesense service isn't showing up in my services listing, it seems like those are not even needed. The integration is automatically adding new sensors upon their first signal sent, and changing state from off to on based on being closed or open, or motion or none.

**Comment by photinus** [comment]
Think this is the same issue we saw in #29

**Comment by jamman9000** [comment]
I'm keeping the version updated, no change to my symptoms yet. Unfortunate the entities don't stay in the registry after a restart, only get added once their state changes.

**Comment by jamman9000** [comment]
Fresh install of HA, HACS, everything and then latest Wyze Sense integration. Same symptoms where the entity will not stay in the registry, they disappear with a HA restart, and the services aren't showing. New error though:

```Thu Nov 14 2019 16:20:43 GMT-0500 (Eastern Standard Time)
Error while setting up platform wyzesense
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/entity_platform.py", line 150, in _async_setup_platform
    await asyncio.wait_for(asyncio.shield(task), SLOW_SETUP_MAX_WAIT)
  File "/usr/local/lib/python3.7/asyncio/tasks.py", line 442, in wait_for
    return fut.result()
  File "/usr/local/lib/python3.7/concurrent/futures/thread.py", line 57, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/config/custom_components/wyzesense/binary_sensor.py", line 82, in setup_platform
    ws = beginConn()
  File "</usr/local/lib/python3.7/site-packages/decorator.py:decorator-gen-2>", line 2, in beginConn
  File "/usr/local/lib/python3.7/site-packages/retry/api.py", line 74, in retry_decorator
    logger)
  File "/usr/local/lib/python3.7/site-packages/retry/api.py", line 33, in __retry_internal
    return f()
  File "/config/custom_components/wyzesense/binary_sensor.py", line 80, in beginConn
    return Open(config[CONF_DEVICE], on_event)
  File "/config/custom_components/wyzesense/wyzesense_custom.py", line 561, in Open
    return Dongle(device, event_handler)
  File "/config/custom_components/wyzesense/wyzesense_custom.py", line 297, in __init__
    self._Start()
  File "/config/custom_components/wyzesense/wyzesense_custom.py", line 492, in _Start
    self._Inquiry()
  File "/config/custom_components/wyzesense/wyzesense_custom.py", line 398, in _Inquiry
    resp = self._DoSimpleCommand(Packet.Inquiry())
  File "/config/custom_components/wyzesense/wyzesense_custom.py", line 393, in _DoSimpleCommand
    self._DoCommand(pkt, cmd_handler, timeout)
  File "/config/custom_components/wyzesense/wyzesense_custom.py", line 379, in _DoCommand
    self._SendPacket(pkt)
  File "/config/custom_components/wyzesense/wyzesense_custom.py", line 331, in _SendPacket
    pkt.Send(self.__fd)
  File "/config/custom_components/wyzesense/wyzesense_custom.py", line 106, in Send
    ss = os.write(fd, pkt)
BrokenPipeError: [Errno 32] Broken pipe

**Comment by teankum** [comment]
> Fresh install of HA, HACS, everything and then latest Wyze Sense integration. Same symptoms where the entity will not stay in the registry, they disappear with a HA restart, and the services aren't showing. New error though:
> 
> ```
> Error while setting up platform wyzesense
> Traceback (most recent call last):
>   File "/usr/src/homeassistant/homeassistant/helpers/entity_platform.py", line 150, in _async_setup_platform
>     await asyncio.wait_for(asyncio.shield(task), SLOW_SETUP_MAX_WAIT)
>   File "/usr/local/lib/python3.7/asyncio/tasks.py", line 442, in wait_for
>     return fut.result()
>   File "/usr/local/lib/python3.7/concurrent/futures/thread.py", line 57, in run
>     result = self.fn(*self.args, **self.kwargs)
>   File "/config/custom_components/wyzesense/binary_sensor.py", line 82, in setup_platform
>     ws = beginConn()
>   File "</usr/local/lib/python3.7/site-packages/decorator.py:decorator-gen-2>", line 2, in beginConn
>   File "/usr/local/lib/python3.7/site-packages/retry/api.py", line 74, in retry_decorator
>     logger)
>   File "/usr/local/lib/python3.7/site-packages/retry/api.py", line 33, in __retry_internal
>     return f()
>   File "/config/custom_components/wyzesense/binary_sensor.py", line 80, in beginConn
>     return Open(config[CONF_DEVICE], on_event)
>   File "/config/custom_components/wyzesense/wyzesense_custom.py", line 561, in Open
>     return Dongle(device, event_handler)
>   File "/config/custom_components/wyzesense/wyzesense_custom.py", line 297, in __init__
>     self._Start()
>   File "/config/custom_components/wyzesense/wyzesense_custom.py", line 492, in _Start
>     self._Inquiry()
>   File "/config/custom_components/wyzesense/wyzesense_custom.py", line 398, in _Inquiry
>     resp = self._DoSimpleCommand(Packet.Inquiry())
>   File "/config/custom_components/wyzesense/wyzesense_custom.py", line 393, in _DoSimpleCommand
>     self._DoCommand(pkt, cmd_handler, timeout)
>   File "/config/custom_components/wyzesense/wyzesense_custom.py", line 379, in _DoCommand
>     self._SendPacket(pkt)
>   File "/config/custom_components/wyzesense/wyzesense_custom.py", line 331, in _SendPacket
>     pkt.Send(self.__fd)
>   File "/config/custom_components/wyzesense/wyzesense_custom.py", line 106, in Send
>     ss = os.write(fd, pkt)
> BrokenPipeError: [Errno 32] Broken pipe
> ```

I'm getting the same error

**Comment by jamman9000** [comment]
This is resolved in v0.0.4
